### PR TITLE
Print errors to stderr only

### PR DESCRIPTION
**Pull request by caruccio**
It's a good practice to send errors to stderr only. That way, json output won't be messed up when something goes wrong, like a layer that failed to be push()ed.
Also, merged and extended #31

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -76,26 +76,26 @@ func main() {
...
 	if len(image.FsLayers) == 0 {
-		fmt.Printf("Can't pull fsLayers")
+		fmt.Fprintf(os.Stderr, "Can't pull fsLayers")
 		os.Exit(1)
 	} else {
 		if useJSONOutput {
 			output.LayerCount = len(image.FsLayers)
 		} else {
-			fmt.Printf("Analysing %d layers\n", len(image.FsLayers))
+			fmt.Fprintf(os.Stderr, "Analysing %d layers\n", len(image.FsLayers))
```
Comment (hashmap) [inline main.go]: Starting from here and below - it's the output in text form, should we use `os.Stderr` in this case?

Comment (caruccio) [inline main.go]: Nope. It was my mistake. I'll revert it as soon as sun rises ;)

Comment (caruccio) [inline main.go]: @hashmap fixed. thanks for spotting that.

**Comment by supereagle** [comment]
@caruccio  Please squash into one commit. Thanks.

File: main.go (hunk 1 above)
Comment (supereagle) [inline main.go]: Is this still not fixed? It's not included in [the revertion commit](https://github.com/optiopay/klar/pull/29/commits/3283eee1e2e891b2721d6f08e2e96440161d4c6e).

**Pull request merged**